data/mods/Aftershock/mobs/aliens.json

```diff
@@ -0,0 +1,155 @@
+    "morale": -7,
+    "melee_cut": 0,
+    "dodge": 6,
+    "reproduction": { "baby_monster": "afs_mon_rabbit", "baby_count": 3, "baby_timer": 55 },
+    "harvest": "alien_small_fur",
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "CATTLEFODDER", "PET_WONT_FOLLOW", "WARM" ]
+  },
+  {
+    "id": "afs_runner",
+    "type": "MONSTER",
+    "name": { "str_sp": "runner" },
+    "description": "A large fluffy three-legged herbivore, extremely fast and cowardly.  It usually grazes alone, and relies on its keen senses to stay safe.",
+    "default_faction": "herbivore",
+    "bodytype": "horse",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "386000 ml",
+    "weight": "386 kg",
+    "hp": 120,
+    "speed": 200,
+    "material": [ "flesh" ],
+    "symbol": "H",
+    "color": "brown",
+    "aggression": -99,
+    "morale": -7,
+    "melee_cut": 0,
+    "dodge": 1,
+    "armor_bash": 4,
+    "armor_cut": 1,
+    "armor_bullet": 1,
+    "vision_night": 25,
+    "path_settings": { "max_dist": 10 },
+    "fear_triggers": [ "SOUND", "PLAYER_CLOSE" ],
+    "death_function": [ "NORMAL" ],
+    "harvest": "alien_large_leather",
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "PET_MOUNTABLE", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "afs_grazer",
+    "type": "MONSTER",
+    "name": { "str_sp": "grazer" },
+    "description": "A large furred herbivore, with its heavy head constantly against the ground.  It uses it clawed hooves to shovel up the snow to reach any dead plant life to eat.",
+    "default_faction": "herbivore",
+    "bodytype": "horse",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "386 L",
+    "weight": "386 kg",
+    "hp": 120,
+    "speed": 130,
+    "material": [ "flesh" ],
+    "symbol": "G",
+    "color": "brown",
+    "aggression": 5,
+    "morale": 80,
+    "melee_skill": 6,
+    "melee_dice": 3,
+    "melee_dice_sides": 4,
+    "melee_cut": 0,
+    "dodge": 1,
+    "armor_bash": 4,
+    "armor_cut": 1,
+    "armor_bullet": 1,
+    "vision_night": 7,
+    "path_settings": { "max_dist": 10 },
+    "anger_triggers": [ "HURT", "MATING_SEASON" ],
+    "death_function": [ "NORMAL" ],
+    "baby_flags": [ "AUTUMN" ],
+    "harvest": "alien_large_leather",
+    "biosignature": { "biosig_item": "feces_manure", "biosig_timer": 8 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "PET_MOUNTABLE", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "ATTACKMON" ]
+  },
+  {
+    "id": "afs_mon_hevel",
+    "type": "MONSTER",
+    "name": { "str": "hevel" },
+    "description": "A small creature covered in blue down, scurrying across the ground fast as it digs through the snow with claws buried in its down, looking for food.  It seems to work in packs and is very territorial.",
```

Comment:
"seems to work in packs and is very territorial" is a bit oddly phrased. "and be very territorial", to go along with "seems"? Pack behavior is likely easier to perceive than territoriality.